test/index.js

```diff
@@ -294,7 +294,7 @@ describe('intl', () => {
 				</IntlProvider>
 			);
 
-			expect(root.innerHTML).to.equal('<div><span>FOO!</span></div>');
+			expect(root.innerHTML).to.equal('<div><div class="markup">FOO!</div></div>');
```

Comment:
This is changing from a `<span>` to a `<div>` as the wrapping component, which could have UI implications.  Can you note that in the README, and we should probably make this a breaking change.  Unless preact-markup has an option to surround with a `<span>` tag

Reply:
It doesn't look like there is any option for controlling the tag used by preact-markup: https://github.com/developit/preact-markup/blob/master/src/index.js#L62

It would probably fit well into the release notes for this, and we would probably need to mark this as a breaking change because of how people may be using MarkupText in bad ways.